python/lsst/ts/observatory/control/data/guider_roi_star_catalog.metadata.yaml

```diff
@@ -0,0 +1,61 @@
+# Metadata sidecar file for Guider ROI Monster guide star catalog
+# This file describes the provenance and characteristics of the guide star catalog
+
+# Data provenance - WHO created this data and HOW
+
+data_source: "Monster catalog + Gaia DR3"
+processing_pipeline: "Guiding catalog generation pipeline v1.0"
+created_by: "Shuang Liang"
+curator: "Shuang Liang"
+contact_email: "shuang.liang.cool@gmail.com"
```

Comment:
It's very usual to put personal details (and personal emails!) into open source repos like this.